src/main/kotlin/com/criptext/mail/scenes/emaildetail/EmailDetailActivity.kt

```diff
@@ -115,9 +109,11 @@ class  EmailDetailActivity: BaseActivity() {
         val result = (view as WebView).hitTestResult
 
         when (result.type) {
-            WebView.HitTestResult.IMAGE_TYPE,
-            WebView.HitTestResult.SRC_IMAGE_ANCHOR_TYPE -> {
-                if(result.extra != null && result.extra!!.startsWith("file://")) {
+            HitTestResult.IMAGE_TYPE,
+            HitTestResult.SRC_IMAGE_ANCHOR_TYPE -> {
+                if(result.extra != null
+                        && result.extra!! !in listOf(WebViewUtils.URI_COLLAPSED_IMG, WebViewUtils.URI_OPENED_IMG)
```

Comment:
This is because on emails that had the 'more' button, you were able to hold and pop up the context menu for that image as if it was an inline image, which is not. This prevents that from happening.